### ISSUE TITLE
[BUG] Broken Links in Rancher chart's README

### ISSUE THREAD

**Issue by Priyashetty17** [rancher/rancher#41729]
**Rancher Server Setup:**
- Rancher version: v2.7-f0d4078f4fa75b303ce276c3f4b12fc465b1ba2a-head

**Issue**: Many links in the "chart [README](https://github.com/rancher/rancher/blob/release/v2.7/chart/README.md)" of the Rancher chart are broken.

**Steps to Reproduce:**
1. Install Rancher.
2. Access the Rancher UI.
3. In the Cluster Explorer, Navigate to Apps -> Installed Apps -> Rancher chart's "chart README".

**Expected behavior:**
- The links should reference valid and existing locations.

**Actual behavior:**
- Most of the links are broken leading to non-existing pages, resulting in HTTP 404 errors.

**Here are some of the Broken links in the chart's [README](https://github.com/rancher/rancher/blob/release/v2.7/chart/README.md):** 
- [Rancher Server ](https://rancher.com/docs/rancher/v2.x/en/installation/k8s-install/)
- [HA install instructions](https://rancher.com/docs/rancher/v2.x/en/installation/how-ha-works/)
- [KE](https://rancher.com/docs/rancher/v2.x/en/installation/k8s-install/kubernetes-rke/) 
- [K3s](https://rancher.com/docs/rancher/v2.x/en/installation/k8s-install/kubernetes-rke/)
- [Install the Required CLI Tools](https://rancher.com/docs/rancher/v2.x/en/installation/k8s-install/helm-rancher/#1-install-the-required-cli-tools)
- [Helm version requirements](https://rancher.com/docs/rancher/v2.x/en/installation/options/helm-version)
- [best practices section](https://rancher.com/docs/rancher/v2.x/en/best-practices/deployment-types/)
- [high-availability Kubernetes installation](https://rancher.com/docs/rancher/v2.x/en/installation/how-ha-works/)
- [Single-node](https://rancher.com/docs/rancher/v2.x/en/installation/k8s-install/#optional-installing-rancher-on-a-single-node-kubernetes-cluster)

**Comment by Priyashetty17** [comment]
Re-opening  this issue because there is still a link that directs to a non-existent page. The "Vue Rancher UI (cluster explorer) custom logos" link under "Advanced Options" should be updated to: https://github.com/rancher/dashboard/tree/master/shell/assets/images/pl.

Rancher version: v2.7-160bee82ebdae08b1bbb9fccab030cf2f31c15ad-head

**Comment by Priyashetty17** [comment]
We need a RC to be able to validate this since it involves modifications to the chart. However, I reviewed the PR and  it appears that one of the two instances of the broken link has not been addressed. 

In the "Advanced Options" section, there are two references of the link "**Vue Rancher UI (cluster explorer) custom logos**". While one link has been updated, there is still another link needs to be updated. The instance that requires updating is here: https://github.com/rancher/rancher/blob/release/v2.7/chart/README.md?plain=1#L202
